### DISCUSSION TITLE
Open303 as Synthengine ?

### DISCUSSION THREAD

**Discussion by highsiderr** [SynthstromAudible/DelugeFirmware#1129] [Ideas]
Hello !

If it were possible to implement the open303 synth as engine in the Deluge?
Then the Deluge would be a standalone Acid box...

https://github.com/RobinSchmidt/Open303

**Comment by n3ptun3s** [comment]
More acid. <3

**Comment by highsiderr** [comment]
Hello .

Question to the programmers.

Would that even be possible?

**Reply by sapphire-arches** [reply]
Should be, just a matter of finding the time. We're all doing this as a hobby so things tend to move a little slow 😅

**Reply by highsiderr** [reply]
No, you are fast enough. You are great.
You get great things out of this little box.

I'm just interested in whether it would be technically possible at all because it's a VST.

**Comment by highsiderr** [comment]
This project also uses open303.
Would be nice to have that in the Deluge.

https://github.com/midilab/jc303
